### PR TITLE
Update ReferenceManyField.md

### PR DESCRIPTION
**Pull request by eboss-dev**
field year doesn't exists in the relational model shown in the example, changed all the year to published_at

### CODE REVIEW

**Comment by fzaninotto** [comment]
Thanks. Could you make it a `DateField` instead of a `TextField`?

File: docs/ReferenceManyField.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const AuthorShow = () => (
       <ReferenceManyField label="Books" reference="books" target="author_id">
         <Datagrid>
           <TextField source="title" />
-          <TextField source="year" />
+          <DateField source="published_at" />
```
Comment (fzaninotto) [inline docs/ReferenceManyField.md:66]: You also need to update the import statement above. Same for other snippets.

**Review by fzaninotto — CHANGES_REQUESTED** [review]

**Review by eboss-dev — COMMENTED** [review]
Added the DateField in the import

File: docs/ReferenceManyField.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ You can also use `<ReferenceManyField>` in a list, e.g. to display the authors o
 
 ```jsx
 import * as React from "react";
-import { List, Datagrid, ChipField, ReferenceManyField, SingleFieldList, TextField } from 'react-admin';
+import { List, Datagrid, ChipField, ReferenceManyField, SingleFieldList, TextField, DateField } from 'react-admin';
```
Comment (slax57) [inline docs/ReferenceManyField.md:80]: no need to change the import in this snippet I believe

**Review by slax57 — CHANGES_REQUESTED** [review]
Other than that the changes look good, thanks

File: docs/ReferenceManyField.md (hunk 2 above)
Comment (eboss-dev) [reply docs/ReferenceManyField.md:80]: Updated

**Comment by eboss-dev** [comment]
Should be good now

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
all good, thanks!

**Pull request merged**